### PR TITLE
Test interfaces

### LINKED ISSUE #63: Add end to end test
Add a test that requires Mugshot from index.js

### PR DESCRIPTION
**Pull request by adela-istrate-hs**
Closes #63

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs @valentin-radulescu-hs can you review?

**Comment by valentin-radulescu-hs** [comment]
LGTM

File: tests/integration/interfaces.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+}, {
+  interface: 'PNGProcessor',
+  functions: ['crop']
+}];
+
+describe('Mugshot interfaces', function() {
+  EXPECTED.forEach(function(item) {
+    var currentInterface = item.interface;
+    describe(`Interface ${currentInterface}`, function() {
+
```
Comment (andrei-picus-hs) [inline tests/integration/interfaces.js]: Remove.

File: tests/integration/interfaces.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      it(`should export the ${currentInterface} interface`, function() {
+        expect(interfaces).to.have.property(currentInterface);
+      });
+
+      item.functions.forEach(function(f) {
+        it(`should have the ${f} function`, function() {
+          expect(interfaces[currentInterface][f]).to.be.an.instanceof(Function);
+        });
+      });
+
```
Comment (andrei-picus-hs) [inline tests/integration/interfaces.js]: Remove.

File: tests/integration/interfaces.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  functions: ['exists', 'readFile', 'writeFile', 'mkdir', 'unlink']
+}, {
+  interface: 'PNGProcessor',
+  functions: ['crop']
+}];
+
+describe('Mugshot interfaces', function() {
+  EXPECTED.forEach(function(item) {
+    var currentInterface = item.interface;
+    describe(`Interface ${currentInterface}`, function() {
```
Comment (andrei-picus-hs) [inline tests/integration/interfaces.js]:
```
Mugshot interfaces
  should contain the ${i} interface
  ${i}
    should have the ${f} function
```

**Comment by andrei-picus-hs** [comment]
Finished review.

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs Can I merge?

**Comment by andrei-picus-hs** [comment]
LGTM

**Pull request merged**